docker-compose.yml

```diff
@@ -726,7 +726,7 @@ services:
 
   ocpdb-flask:
     <<: *ocpdb-defaults
-    command: ["gunicorn", "--bind", "0.0.0.0:5000", "webapp.entry_point_gunicorn:app"]
+    command: ["gunicorn", "--workers", "16", "--bind", "0.0.0.0:5000", "webapp.entry_point_gunicorn:app"]
```

Comment:
As these service are not running exclusively on the host, shouldn't we better make these configurable?

Reply:
Good idea, done.